qrScanner/src/main/java/io/github/novacrypto/qrscanner/ScanQrActivity.kt

```diff
@@ -34,6 +34,7 @@ open class ScanQrActivity : AppCompatActivity() {
     companion object {
         const val OPTION_SHOW_BARCODE_BOX = "SHOW_BARCODE_BOX"
         const val BARCODE_DATA = "BARCODE_DATA"
+        const val AUTOACCEPT_RESULT = "AUTOACCEPT_RESULT"
```

Comment:
This would be nice if matched the existing option:
```
const val OPTION_AUTOACCEPT_RESULT = "AUTOACCEPT_RESULT"
```
And put them side by side.